include/DDpackage.h

```diff
@@ -46,7 +46,6 @@ namespace dd {
     constexpr unsigned short TTMASK            = TTSLOTS - 1; // must be TTSLOTS-1
     constexpr unsigned int   NODE_CHUNK_SIZE   = 2000;        // this parameter may be increased for larger benchmarks to minimize the number of allocations
     constexpr unsigned int   LIST_CHUNK_SIZE   = 2000;
-    constexpr unsigned short MAXN              = 128; // max no. of inputs
```

Comment:
Would it make sense to keep a max number of qubits to warn users if they are about to jump in at the deep end?

Reply:
How high would you set such a number? Entanglement benchmarks can be performed up to the thousands of qubits.